### PR TITLE
Change the title of configure severity to resources file

### PR DESCRIPTION
**Pull request by Cosifne**
Fix https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1431609/

### CODE REVIEW

File: src/Features/Core/Portable/CodeFixes/Configuration/ConfigureSeverity/ConfigureSeverityLevelCodeFixProvider.cs [hunk 1]
```diff
@@ -24,11 +24,11 @@ internal sealed partial class ConfigureSeverityLevelCodeFixProvider : IConfigura
     {
         private static readonly ImmutableArray<(string name, string value)> s_editorConfigSeverityStrings =
```
Comment (jasonmalinowski) [inline src/Features/Core/Portable/CodeFixes/Configuration/ConfigureSeverity/ConfigureSeverityLevelCodeFixProvider.cs:25]: The tuple elements "name" and "value" here seems quite confusing -- if anything I would have expected them to be flipped around where the value is the one being put into the file. Maybe call one "display string" or something so it's very clear it's for display only?

File: src/Features/Core/Portable/CodeFixes/Configuration/ConfigureSeverity/ConfigureSeverityLevelCodeFixProvider.cs [hunk 2]
```diff
@@ -24,11 +24,11 @@ internal sealed partial class ConfigureSeverityLevelCodeFixProvider : IConfigura
...
             ImmutableArray.Create(
-                (nameof(EditorConfigSeverityStrings.None), EditorConfigSeverityStrings.None),
-                (nameof(EditorConfigSeverityStrings.Silent), EditorConfigSeverityStrings.Silent),
-                (nameof(EditorConfigSeverityStrings.Suggestion), EditorConfigSeverityStrings.Suggestion),
-                (nameof(EditorConfigSeverityStrings.Warning), EditorConfigSeverityStrings.Warning),
-                (nameof(EditorConfigSeverityStrings.Error), EditorConfigSeverityStrings.Error));
+                (nameof(EditorConfigSeverityStrings.None), FeaturesResources.None),
```
Comment (jasonmalinowski) [inline src/Features/Core/Portable/CodeFixes/Configuration/ConfigureSeverity/ConfigureSeverityLevelCodeFixProvider.cs:27]: This doesn't seem right -- at least one of these should be the value that goes into the file, right, which is the lowercase strings?

Comment (Cosifne) [reply src/Features/Core/Portable/CodeFixes/Configuration/ConfigureSeverity/ConfigureSeverityLevelCodeFixProvider.cs:27]: Yep, I just notice that case. fixing it now

Comment (Cosifne) [reply src/Features/Core/Portable/CodeFixes/Configuration/ConfigureSeverity/ConfigureSeverityLevelCodeFixProvider.cs]: OK, fixed

File: src/Features/Core/Portable/CodeFixes/Configuration/ConfigureSeverity/ConfigureSeverityLevelCodeFixProvider.cs [hunk 3]
```diff
@@ -58,11 +58,11 @@ private static ImmutableArray<CodeFix> GetConfigurations(Project project, IEnume
...
                 var nestedActions = ArrayBuilder<CodeAction>.GetInstance();
-                foreach (var (name, value) in s_editorConfigSeverityStrings)
+                foreach (var (name, value, title) in s_editorConfigSeverityStrings)
                 {
                     nestedActions.Add(
                         new SolutionChangeAction(
-                            name,
+                            title,
                             solution => ConfigurationUpdater.ConfigureSeverityAsync(value, diagnostic, project, cancellationToken),
                             name));
```
Comment (jasonmalinowski) [inline src/Features/Core/Portable/CodeFixes/Configuration/ConfigureSeverity/ConfigureSeverityLevelCodeFixProvider.cs:67]: Is name only being used here for the equivalence key? I'd imagine we can just use the value then?

Comment (Cosifne) [reply src/Features/Core/Portable/CodeFixes/Configuration/ConfigureSeverity/ConfigureSeverityLevelCodeFixProvider.cs:67]: Are you saying we can get rid of the 'name' here?
Yes that possible

Comment (Cosifne) [reply src/Features/Core/Portable/CodeFixes/Configuration/ConfigureSeverity/ConfigureSeverityLevelCodeFixProvider.cs]: Resolved

File: src/Features/Core/Portable/FeaturesResources.resx [hunk 4]
```diff
@@ -2945,4 +2945,19 @@ Zero-width positive lookbehind assertions are typically used at the beginning of
...
+  </data>
+  <data name="Silent" xml:space="preserve">
+    <value>Silent</value>
+  </data>
+  <data name="Suggestion" xml:space="preserve">
+    <value>Suggestion</value>
+  </data>
+  <data name="Warning" xml:space="preserve">
+    <value>Warning</value>
+  </data>
```
Comment (jasonmalinowski) [inline src/Features/Core/Portable/FeaturesResources.resx:2962]: We do already have these in the WorkspacesResoruces.resx; not sure if duplicating it here creates extra work for the loc team or not.

**Review by jasonmalinowski — APPROVED** [review]
Looks good, but we can probably reuse the existing strings.

File: src/Features/Core/Portable/FeaturesResources.resx (hunk 4 above)
Comment (Cosifne) [reply src/Features/Core/Portable/FeaturesResources.resx]: I have changed to resue that. but still the loc team needs to translate the 'slient' word

**Pull request merged**